cmd/config.go

```diff
@@ -164,6 +166,96 @@ keyfiles written in:
 	},
 }
 
+var encryptCredentialsDataCmd = &cobra.Command{
+	Use:     "encrypt-data <encoding keyfile> <text>",
+	Aliases: []string{"encrypt-data"},
+	Short:   "Encrypt data with the specified encryption keyfile.",
+	Long: `
+cozy-stack config encrypt-data encrypts any valid JSON data
+
+example :
+./cozy-stack config encrypt-data ~/.cozy/key.enc "{\"foo\": \"bar\"}"
```

Comment:
It looks like `cobra.Command` now has an [Example](https://godoc.org/github.com/spf13/cobra#Command) field. We should try to use it